docs/command-line/gwin.rst

```diff
@@ -54,8 +52,24 @@ Configuration file syntax
 -------------------------
 
 Configuration files follow the ``ConfigParser`` syntax.
-There are two required sections.
-One is a ``[variable_args]`` section that contains a list of parameters that we will vary to obtain a posterior distribution and the other is ``[static_args]`` section that contains a list of parameters are held fixed through out the run.
+There are three required sections.
+
+One is a ``[likelihood]`` section that contains the name of the likelihood class
+to use for evaluating the posterior. An example::
+
+    [likelihood]
+    name = gaussian
+
+In this case, the :py:class:`gwin.likelihood.GaussianLikelihood` would be used.
+Examples of using this likelihood class on a BBH injection and on GW150914 are
+given below. Any name that starts with ``test_`` is an analytic test
```

Comment:
While not really within this PR, we could add here an example on how to use simulated noise?

Reply:
The BBH example is on simulated noise. See lines 396 and 397 of `gwin.rst`.